src/routes/admin.routes.ts

```diff
@@ -15,7 +15,7 @@ router.post("/office/login", AdminLoginHandler);
 // router.get("/office/dashboard", validateToken, AdminDashboardHandler);
 router.get("/office", validateToken, GetAllAdminHandler);
 // router.post("/office/new", validateToken, CreateAdminHandler);
-// router.post("/office/edit/{adminId}", validateToken, EditAdminHandler);
+router.post("/office/edit/:adminId", validateToken, EditAdminHandler);
```

Comment:
Remove `adminId` as we are not using it. We are relying on the mail being sent in the body of the request.

Reply:
what should come here?
office/edit/email?

Reply:
No, do not extract the mail from params. Get it from the body.